src/event.js

```diff
@@ -677,30 +683,24 @@ jQuery.Event.prototype = {
 	isImmediatePropagationStopped: returnFalse,
 
 	preventDefault: function() {
-		var e = this.originalEvent;
-
 		this.isDefaultPrevented = returnTrue;
 
-		if ( e && e.preventDefault ) {
-			e.preventDefault();
+		if ( this.originalEvent ) {
```

Comment:
This loses the guard for `preventDefault` and `stopPropagation` existing before calling them. It makes me nervous but I cannot think of situations where that would be a problem. For example, if these are nested `jQuery.Event` instances (*really* common in jQuery UI) it should work fine.